### PR TITLE
avoid nix/default.nix not found error

### PR DESCRIPTION
**Pull request by mmsqe**

### CODE REVIEW

File: pystarport/default.nix [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{ pkgs ? import ../../nix { }, commit ? "" }:
+{ pkgs ? import <nixpkgs> {}, commit ? "" }:
```
Comment (yihuang) [inline pystarport/default.nix:1]: This file should be removed I think

Comment (mmsqe) [reply pystarport/default.nix:1]: you mean we dont need import?

Comment (yihuang) [reply pystarport/default.nix:1]: don't need this file at all

Comment (mmsqe) [reply pystarport/default.nix:1]: why rm pystarport/default.nix? I find it helpful to [gen](https://github.com/crypto-org-chain/cronos/pull/625/commits/976bb7214b9af9f5640f64f690c16b152b77e5ef) proto

Comment (yihuang) [reply pystarport/default.nix:1]: I see

**Review by yihuang — APPROVED** [review]

**Pull request merged**